docs/codeql/reusables/vs-code-basic-instructions/run-quick-query-1.rst

```diff
@@ -0,0 +1,7 @@
+The CodeQL extension for Visual Studio Code adds several **CodeQL:** commands to the command palette including **Quick Query**, which you can use to run a query without any set up.
```

Comment:
Not sure about this one, you are really the experts here!
```suggestion
The CodeQL extension for Visual Studio Code adds several **CodeQL:** commands to the command palette including **Quick Query**, which you can use to run a query without any set-up.
```

Reply:
Thanks for checking. This is usually not hyphenated in US ENglish.

Reply:
@felicitymay Isn't `setup` a single word when used as noun?

Reply:
I agree Arthur, I think this sentence is worded in such a way that it's not clear whether the term is intended to be a noun or a verb. I suggest we make the wording more explicit.

```suggestion
The CodeQL extension for Visual Studio Code adds several **CodeQL:** commands to the command palette including **Quick Query**, which you can use to create and run a query without setting up a CodeQL pack.
```